cmd/geth/config.go

```diff
@@ -236,5 +236,12 @@ func applyMetricConfig(ctx *cli.Context, cfg *gethConfig) {
 }
 
 func deprecated(field string) bool {
-	return false
+	switch field {
+	case "ethconfig.Config.EVMInterpreter":
+		return true
+	case "ethconfig.Config.EWASMInterpreter":
+		return true
+	default:
+		return false
```

Comment:
Are you sure this is the 'namespace' they wind up in? I tried a `geth --vm.ewasm=foo dumpconfig > foo.conf`, and they wound up like this: 

```
[Eth]
EWASMInterpreter = "foo"
EVMInterpreter = ""
```

Reply:
Yeah the name is coming from `reflect.Type.Name()` of each config struct, which uses the `packageName.typeName` as its format. This is what I get when I use a similar config as your example:

```
WARN [07-06|09:28:46.630] Config field is deprecated and won't have an effect name=ethconfig.Config.EVMInterpreter
```